vars/Pipeline.groovy

```diff
@@ -287,6 +310,14 @@ def call() {
                 AWS_SECRET_ACCESS_KEY = credentials('AWS_SECRET_ACCESS_KEY')
                 tomcatUsername = credentials('TOMCAT_USERNAME')
                 tomcatPassword = credentials('TOMCAT_PASSWORD')
+
+                WUM_UAT_URL=credentials('WUM_UAT_URL')
```

Comment:
Why is the property reading duplicated? I see the same in line 79

Reply:
ATM this pipeline has been modified by @pasindujw to support his parallel execution changes. So i added the same code changes to both the conditions.